core/object/object.cpp

```diff
@@ -36,6 +36,8 @@
 #include "core/io/resource.h"
 #include "core/object/class_db.h"
 #include "core/object/message_queue.h"
+#include "core/object/method_info.h"
+#include "core/object/property_info.h"
```

Comment:
They're included in the `.h`, so shouldn't be needed here I suppose?
And I see `message_queue.h` is also duplicated.

Reply:
Good catch on the duplicated includes! I'll rebase this today :)